### PR TITLE
New tests

### PR DESCRIPTION
**Pull request by MorganLove**
**Description**

- Re-activating and updating tests, adding new mocks, updating snapshots

**Testing**

- Yup

**Progress**

- Should be clear to merge these. I'd rather do these test updates in smaller batches so they don't get super massive, and so they can keep up with the other changes going on in the codebase.

### CODE REVIEW

File: client/src/components/MultiJurisdictionAudit/useSetupMenuItems/_mocks/index.ts [hunk 1]
```diff
@@ -503,8 +531,8 @@ export const fileProcessingMocks: {
   null: null,
   processed: {
     status: FileProcessingStatus.PROCESSED,
-    startedAt: 'sometime',
-    completedAt: 'a different time',
+    startedAt: 'sometim',
+    completedAt: 'a different tim',
```
Comment (jonahkagan) [inline client/src/components/MultiJurisdictionAudit/useSetupMenuItems/_mocks/index.ts:535]: maybe make these actual timestamps?

**Review by jonahkagan — APPROVED** [review]

File: client/src/components/MultiJurisdictionAudit/useSetupMenuItems/_mocks/index.ts (hunk 1 above)
Comment (MorganLove) [reply client/src/components/MultiJurisdictionAudit/useSetupMenuItems/_mocks/index.ts:535]: Gah, I thought I caught all those before, haha. Good catch.

File: client/src/components/MultiJurisdictionAudit/__snapshots__/index.test.tsx.snap [hunk 2]
```diff
@@ -503,356 +1139,360 @@ exports[`AA setup flow renders sidebar when authenticated on /setup 1`] = `
...
+                class="bp3-html-select sc-jTzLTM hmeDmq"
               >
-                <svg
-                  data-icon="double-caret-vertical"
-                  height="16"
-                  viewBox="0 0 16 16"
-                  width="16"
+                <select
+                  data-testid="state-field"
+                  field="[object Object]"
```
Comment (jonahkagan) [inline client/src/components/MultiJurisdictionAudit/__snapshots__/index.test.tsx.snap:1155]: @MorganLove this looks bad

File: client/src/components/MultiJurisdictionAudit/__snapshots__/index.test.tsx.snap [hunk 3]
```diff
@@ -1134,32 +1786,43 @@ exports[`JA setup submits ballot manifest 1`] = `
...
+              </strong>
+               
+              manifest.csv
+            </p>
+            <p
+              class="sc-iwsKbI hwKioB"
+            >
+              Upload successfully completed at
+               
+              Invalid Date
```
Comment (jonahkagan) [inline client/src/components/MultiJurisdictionAudit/__snapshots__/index.test.tsx.snap:1803]: @MorganLove this doesn't seem right

**Pull request merged**